### DISCUSSION TITLE
bypass blocks micrsoft labs

### DISCUSSION THREAD

**Discussion by abditube1345** [3kh0/ext-remover#866] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
install firefox

**Comment by Aka-but-nice** [comment]
Yes.

**Reply by Aka-but-nice** [reply]
I totally agree with this, Firefox on top. Huge W, W Rizz, and last, but not least, skibidi toilet. That'll get them!

**Comment by abditube1345** [comment]
an mistake it blocked websites after updates like vpn and discord the app that blocked is "Fortinet"

**Reply by Aka-but-nice** [reply]
I totally agree with this, Firefox on top. Huge W, W Rizz, and last, but not least, skibidi toilet. That'll get them!

**Comment by abditube1345** [comment]
to bypass  this go to extension on vm and find chrome web store and search vpn
